cha-ching-app/src/CheckoutForm.js

```diff
@@ -1,36 +1,42 @@
-import { PaymentElement, useStripe, useElements } from "@stripe/react-stripe-js";
+import {
+  PaymentElement,
+  useStripe,
+  useElements,
+} from "@stripe/react-stripe-js";
 
 function CheckoutForm() {
-    const stripe = useStripe()
-    const elements = useElements()
+  const stripe = useStripe();
+  const elements = useElements();
 
-    const handleSubmit = async () => {
-        await stripe.confirmPayment({
-            elements,
-            confirmParams: {
-                return_url: "http://localhost:3000/success",
-            },
-        })
-        .then(data => console.log("What am I (data):", data))
-        .catch(err => console.log("Something went wrong confirming the payment:", err))
-    }
+  const handleSubmit = async () => {
+    await stripe
+      .confirmPayment({
+        elements,
+        confirmParams: {
+          return_url: "http://localhost:3000/success",
+        },
+      })
+      .then((data) => console.log("What am I (data):", data))
+      .catch((err) =>
+        console.log("Something went wrong confirming the payment:", err),
+      );
+  };
 
-    return (
-      <div>
-        <div className="header">Cha-ching</div>
-        <div className="paymentFormContainer">
-          <form id="payment-form">
-            <div className="paymentElementcontainer">
-                <PaymentElement />
-            </div>
-            
-          </form>
-          <button onClick={handleSubmit} className="button">
-              Submit
-            </button>
-        </div>
+  return (
+    <div>
+      <div className="header">Cha-ching</div>
+      <div className="paymentFormContainer">
+        <form id="payment-form">
+          <div className="paymentElementcontainer">
+            <PaymentElement />
+          </div>
+        </form>
```

Comment:
This form might not be needed, in the tutorials we saw it was being used to associate the button that's inside the form to the payment element but we ended up bringing that button outside the form. Lets try removing it and see if it behaves the same